Tests/Tests/RouterSpec.swift

```diff
@@ -8,14 +8,19 @@ import RSDKUtilsNimble
 import class RSDKUtilsMain.TypedDependencyManager
 #endif
 @testable import RInAppMessaging
+@testable import NIO
```

Comment:
nit: What is NIO?

Reply:
It's `Shock`'s dependency - SwiftNIO (non-blocking I/O). I think it's a leftover from some code. I can't remember what it was for. I'll just remove it